.github/workflows/release.yaml

```diff
@@ -69,8 +69,15 @@ jobs:
           - os: windows-latest
             target: x86_64-pc-windows-msvc
             flags: --features=native-tls
+            rustflags: -C target-feature=+crt-static
     steps:
       - uses: actions/checkout@v2
+
+      - name: Set RUSTFLAGS env variable
+        if: matrix.job.rustflags
+        shell: bash
```

Comment:
Unless I set `shell` to `bash`, `RUSTFLAGS` will not get passed to cargo in the next step.